src/containers/Login.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { Button, FormGroup, FormControl, ControlLabel } from "react-bootstrap";
 import "./Login.css";
-import * as api from "../api/base";
+import * as token from "../api/token";
```

Comment:
Maybe token is a bad name?

Assumes how we're logging in.... Maybe "session" is a better word? Hmmm....